_data/matrix/Passport.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Passport"
+category: "Signing Device"
+keyfeatures: "Open-source, intuitive, beautiful, and highly secure self-custody."
+homepage: "https://foundation.xyz/passport"
+
+feature:
+  device: ""
+  os: ""
+  web: "Not Applicable"
+  hw_interface: "No"
```

Comment:
Absolute novice talking my way through here:  If HWI is technically a "_[Python library and command-line tool used to interface with hardware wallets_](https://bitcoinops.org/en/topics/hwi/)", then I would expect HWI to be "Not Applicable" for Signing Devices (as you suggest). 

Would you agree, is that a good way to put this point to rest?

Reply:
Oh I see. I probably misunderstood that column. I thought `hw_interface` was generally referring to whether a software wallet can interact with a hardware signer, but now I get the sense that it specifically refers to whether something is compatible with the [HWI](https://bitcoinops.org/en/topics/hwi/) library.

If that is the intended meaning, it might make sense to call the YAML field just `hwi`, at least that threw me off. Perhaps the tooltip for that column could also be changed in that regard, I’ll suggest something there.